README.md

```diff
@@ -53,6 +53,23 @@ rend.BuildPage(w, mappedPageData, "name-of-template-file-without-extension")
 
 If an error occurs during page build, either because of an incorrect template name or incorrect data mapping, `dp-renderer` will write an error via an `errorResponse` struct.
 
+### Using the ONS Design System
+
+`dp-renderer` supports use of the [ONS Design System](https://ons-design-system.netlify.app/) on a **per-page basis**. In order to use the ONS Design System, the page mapper function should set the version number to the `FeatureFlags.ONSDesignSystemVersion` variable in the core page model struct, like so:
```

Comment:
Think this needs to be clearer that the page mapper function is in the calling app rather than the renderer library. Something like, "...to use the ONS Design System, the page mapper function in the calling application should set the version..."